package.json

```diff
@@ -1,75 +1,79 @@
 {
-  "name": "react-list-select",
```

Comment:
Please don't reformat `package.json`; it makes it a pain whenever npm wants to do something.